worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitPlayer.java

```diff
@@ -67,26 +67,28 @@
 import java.util.Map;
 import java.util.Locale;
 import java.util.UUID;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
 
 import javax.annotation.Nullable;
 
 public class BukkitPlayer extends AbstractPlayerActor {
 
-    private Player player;
-    private WorldEditPlugin plugin;
-    private PermissionAttachment permAttachment;
+    private static final PermAttachmentManager permAttachmentManager = new PermAttachmentManager(WorldEditPlugin.getInstance());
```

Comment:
is it not better worth having this in the WorldEditPlugin itself, and the manager itself being a separate class, and then accessing with WorldEditPlugin.getInstance().getPermAttachmentManager() ?

Reply:
I don't think it makes a huge difference since you don't really use it anywhere else I think?

Reply:
i don't, we only ever use it within the class and the perm attachment is not accessible from the outside

Reply:
I understand the reluctance to call `WorldEditPlugin.getInstance()` but considering that this is for a Bukkit wrapper class I think this is fine.

Reply:
done